frontend/src/hooks/usePlausibleTracker.ts

```diff
@@ -66,7 +66,8 @@ export type CustomEvents =
     | 'new-in-unleash-click'
     | 'new-in-unleash-dismiss'
     | 'search-opened'
-    | 'events-exported';
+    | 'events-exported'
+    | 'timeline-usage';
```

Comment:
Just so we don't risk any overlaps with other eventual timelines in Unleash? Doesn't hurt to be a bit more specific.

```suggestion
    | 'event-timeline-usage';
```

Reply:
Thank you, good catch!